src/status_im/transport/message/core.cljs

```diff
@@ -81,9 +81,7 @@
         (fx/merge cofx
                   (process-next response-js sync-handler)
                   (models.contact/ensure-contacts
-                   (map data-store.contacts/<-rpc contacts-clj)
-                   (when chats
-                     (map data-store.chats/<-rpc (types/js->clj chats))))))
+                   (map data-store.contacts/<-rpc contacts-clj) chats)))
```

Comment:
could you elaborate, i don't see changes in ensure-contacts, how it will work ?

Reply:
actually ensure-contacts is just a middle man, chats are only used in block-contact function of block.cljs

Reply:
https://github.com/status-im/status-react/pull/12878/files#diff-bec107b652230a20367afcf97c75c0d66b35f99cbca6484d59a9ddd17f4422a5

Reply:
i see